src/Extension/AbstractOrderedExtension.php

```diff
@@ -33,10 +33,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     /**
      * {@inheritdoc}
      */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    public function configureOptions(OptionsResolver $resolver)
```

Comment:
Just add `configureOptions()` and call it from `setDefaultOptions()` (you can remove the legacy one when you drop support for Symfony 2.7 in the future).